python/nano/src/bigdl/nano/pytorch/runtime_binding/openvino_inference.py

```diff
@@ -0,0 +1,183 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from pytorch_lightning import LightningModule
+import torch
+from torch.utils.data import DataLoader
+from bigdl.nano.pytorch.lightning import LightningModuleFromTorch
+from functools import partial, wraps
+import warnings
+import torch
+import numpy as np
+import inspect
+from openvino.inference_engine import IECore
+
+
+OPENVINO_BINDED_COMPONENTS = ['ir_up_to_date', 'ir_model']
```

Comment:
is there a way for user to export and save the openvino model?

Reply:
Users can export the model by `model.eval_openvino(inputs, filepath)`.  
They can find the saved model in `filepath`.

Reply:
this is not very intuitive; we may add an export method in model or trainer?

Reply:
Let's do this in another issue.   
https://github.com/intel-analytics/BigDL/issues/4313